core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -1285,7 +1285,6 @@ ExWorkProcRetcode ExHbaseAccessBulkLoadPrepSQTcb::work()
       matches_ = 0;
       currRowNum_ = 0;
       numRetries_ = 0;
-      hFileParamsInitialized_ = FALSE;
```

Comment:
It looks like the sampling is controlled by (if samplingRate > 0). I guess you mean flush to add a batch of rows to HFile in one shot. Otherwise, I thought that flushing of hfile should happen when the hfile is closed. Is that correct?

Please remove the comment.  Then this fix is good to go and it is definitely a great find to improve the bulk load further.